alfred-telemetry-solr/README.md

```diff
@@ -105,77 +105,77 @@ At the moment following metrics are included, for each core tracked by solr:
 At the moment following metrics are included, only for core "alfresco":
 
 | Name                                                                  |
 | ----------------------------------------------------------------------|
-| solr_alfresco__afts_handlerStart                                      |
-| solr_alfresco__afts_requests                                          |
-| solr_alfresco__afts_timeouts                                          |
-| solr_alfresco__afts_errors                                            |
-| solr_alfresco__afts_clientErrors                                      |
-| solr_alfresco__afts_serverErrors                                      |
-| solr_alfresco__afts_75thPcRequestTime                                 |
-| solr_alfresco__afts_95thPcRequestTime                                 |
-| solr_alfresco__afts_99thPcRequestTime                                 |
-| solr_alfresco__afts_999thPcRequestTime                                |
-| solr_alfresco__afts_medianRequestTime                                 |
-| solr_alfresco__afts_avgTimePerRequest                                 |
-| solr_alfresco__afts_totalTime                                         |
-| solr_alfresco__afts_5minRateRequestsPerSecond                         |
-| solr_alfresco__afts_15minRateRequestsPerSecond                        |
-| solr_alfresco__afts_avgRequestsPerSecond                              |
-| solr_alfresco__cmis_handlerStart                                      |
-| solr_alfresco__cmis_requests                                          |
-| solr_alfresco__cmis_timeouts                                          |
-| solr_alfresco__cmis_errors                                            |
-| solr_alfresco__cmis_clientErrors                                      |
-| solr_alfresco__cmis_serverErrors                                      |
-| solr_alfresco__cmis_75thPcRequestTime                                 |
-| solr_alfresco__cmis_95thPcRequestTime                                 |
-| solr_alfresco__cmis_999thPcRequestTime                                |
-| solr_alfresco__cmis_medianRequestTime                                 |
-| solr_alfresco__afts_avgTimePerRequest                                 |
-| solr_alfresco__cmis_totalTime                                         |
-| solr_alfresco__cmis_5minRateRequestsPerSecond                         |
-| solr_alfresco__cmis_15minRateRequestsPerSecond                        |
-| solr_alfresco__cmis_avgRequestsPerSecond                              |
-| solr_alfresco_searcher_warmupTime                                     |
-| solr_alfresco_searcher_deletedDocs                                    |
-| solr_alfresco_searcher_maxDoc                                         |
-| solr_alfresco_searcher_numDocs                                        |
-| solr_alfresco_searcher_indexVersion                                   |
+| solr.alfresco./afts.handlerStart                                      |
```

Comment:
what is the `/` for ?

Reply:
Those are names, attributes, properties of the beans, with no manipulation. 
Prometheus registry takes care of escaping etc.